sdk/containerregistry/azure-containerregistry/azure/containerregistry/_anonymous_exchange_client.py

```diff
@@ -36,25 +36,25 @@ def __init__(self, endpoint, **kwargs):  # pylint: disable=missing-client-constr
         )
 
     def get_acr_access_token(self, challenge, **kwargs):
-        # type: (str, Dict[str, Any]) -> str
+        # type: (str, Any) -> str
         parsed_challenge = _parse_challenge(challenge)
         parsed_challenge["grant_type"] = TokenGrantType.PASSWORD
         return self.exchange_refresh_token_for_access_token(
-            None,
+            "",
             service=parsed_challenge["service"],
             scope=parsed_challenge["scope"],
             grant_type=TokenGrantType.PASSWORD,
             **kwargs
         )
 
     def exchange_refresh_token_for_access_token(
-        self, refresh_token=None, service=None, scope=None, grant_type=TokenGrantType.PASSWORD, **kwargs
+        self, refresh_token, service, scope, grant_type, **kwargs
     ):
-        # type: (str, str, str, str, Dict[str, Any]) -> str
+        # type: (str, str, str, Union[str, TokenGrantType], Any) -> str
         access_token = self._client.authentication.exchange_acr_refresh_token_for_acr_access_token(
             service=service, scope=scope, refresh_token=refresh_token, grant_type=grant_type, **kwargs
         )
-        return access_token.access_token
+        return access_token.access_token # type: ignore
```

Comment:
You shouldn't need to ignore this - is `self._client.authentication.exchange_acr_refresh_token_for_acr_access_token` type hinted correctly?